### PR TITLE
Create head_or_tails.py

### PR DESCRIPTION
**Pull request by lucaytpols**
## Summary

This is a pull request for a new example!

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/head_or_tails.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import random
+
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="h!")
```
Comment (H4ckerxx44) [inline examples/head_or_tails.py:5]: I would suggest '$' as a prefix for consistency

Comment (lucaytpols) [reply examples/head_or_tails.py:5]: Okay! I added some fixes tho.

File: examples/head_or_tails.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    number = random.randint(0,1)
+
+    tails = 0
+    heads = 1
+
+    if answer is None:
+        await ctx.send("You have not set an argumeny, please do: h!headsortails `heads` or h!headsortails `tails`")
+    else:
+        if answer == "heads" or answer == "tails":
+            if number == tails:
```
Comment (TAG-Epic) [inline examples/head_or_tails.py:18]: It would be better to hardcode the values here

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
Woudnt it be better to just do something like https://hst.sh/emihemovil.makefile

File: examples/head_or_tails.py (hunk 2 above)
Comment (lucaytpols) [reply examples/head_or_tails.py:18]: what you want! :)

Comment (lucaytpols) [reply examples/head_or_tails.py]: I added the things you wanted. I hope it's good now!

File: examples/head_or_tails.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import random
+
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="$")
+
+@bot.command()
+async def headortails(ctx, answer=None:
```
Comment (TAG-Epic) [inline examples/head_or_tails.py:8]: You are missing a ) here

Comment (lucaytpols) [reply examples/head_or_tails.py:8]: I am so sorry If i make so many mistakes :(

File: examples/head_or_tails.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    machine_choice = random.choice(["heads", "tails"])
+
+    if machine_choice == answer:
```
Comment (veni-vidi-code) [inline examples/head_or_tails.py:11]:
```suggestion
    if random.choice(["heads", "tails"]) == answer:
```

Comment (lucaytpols) [reply examples/head_or_tails.py]: Added!

File: examples/head_or_tails.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+        await ctx.send("Congratulations")
+    else:
+        await ctx.send("Sorry you lost")
```
Comment (veni-vidi-code) [inline examples/head_or_tails.py:13]:
```suggestion
        await ctx.reply("Congratulations")
    else:
        await ctx.reply("Sorry you lost")
```

Comment (lucaytpols) [reply examples/head_or_tails.py:13]: Is that a new syntax? 🤔

Comment (lucaytpols) [reply examples/head_or_tails.py]: Added but still confused why it changed to reply.

File: examples/head_or_tails.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import random
+
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="$")
+
+@bot.command()
+async def headortails(ctx, answer=None):
+
```
Comment (TAG-Epic) [inline examples/head_or_tails.py:9]: This space should be removed. Also possibly add some comments to the top of the file to explain what it does?

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: examples/head_or_tails.py (hunk 6 above)
Comment (lucaytpols) [reply examples/head_or_tails.py:9]: Okay!

Comment (lucaytpols) [reply examples/head_or_tails.py]: Done.

**Review by TAG-Epic — APPROVED** [review]
LGTM

**Pull request merged**